Source/WebCore/dom/StyledElement.cpp

```diff
@@ -192,6 +193,11 @@ void StyledElement::invalidateStyleAttribute()
 
     Node::invalidateStyle(validity);
 
+    if (isSVGElement()) {
+        if (auto* svgElement = dynamicDowncast<SVGElement>(this))
+            SVGElement::InstanceInvalidationGuard guard(*svgElement);
+    }
+
```

Comment:
You can just call `invalidateInstances()` directly instead of creating a pointless InstanceInvalidationGuard

Reply:
made `invalidateInstances` public